docs/release-notes/v2.26/index.md

```diff
@@ -0,0 +1,64 @@
+# November.1 2020 (version 2.26)
+
+---
+
+Welcome to the November.1 2020 release of Gatsby (version 2.26). Key highlights of this release:
+
+- [New Image Plugin](#new-gatsby-image-beta) - access static images without GraphQL, high Lighthouse scores again
+- [File System Route API](#file-system-route-api) - create pages using filename conventions
+- [New Release Process](#new-release-process) - more stability and clarity around Gatsby releases
+- [gatsby-source-contentful v4.0](#gatsby-source-contentful) - new RichText implementation
+
+Other notable changes:
+
+- [Enable compression in the dev server](#enable-compression-in-the-dev-server) - speeds up remote previews
+- [gatsby-plugin-mdx](#gatsby-plugin-mdx) - performance improvements
+- [Notable bug-fixes](#)
+
+Sneak peek to next releases:
+
+- [Queries on Demand](#) - quicker `develop` startup for big sites
+- [create-gatsby](#) - better `gatsby new`
+- [Experimental: Lazy images in develop](#)
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know
+if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+### New Gatsby Image (Beta)
+
+TODOC
+
+- [README](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-image)
+- [Discussion](https://github.com/gatsbyjs/gatsby/discussions/27950)
+
+### File System Route API
+
+TODOC
+
+### New Release Process
+
+TODOC
+
+### Enable compression in the dev server
+
+TODOC
+
+### gatsby-source-contentful
+
+TODOC
+
+### gatsby-plugin-mdx
+
+TODOC
```

Comment:
```suggestion
- [#27937](https://github.com/gatsby/gatsby/issues/27937): There was a significant performance improvement by making a certain node fetching step lazy. This is especially noticeable at scale.

- [#27941](https://github.com/gatsby/gatsby/issues/27941): Adds a `lessBabel` option to the plugin which allows you to use a fast path for scanning exports during sourcing. The savings are significant, especially at scale, but as the new scanner does not use Babel, if your site depends on Babel then you can't use this. Please give the option a try and report any problems unrelated to not running Babel so we can improve the support.
```

Reply:
`lessBabel` PR goes to the next release. Added a note about the other one.